src/v/kafka/requests/requests.cc

```diff
@@ -126,6 +128,10 @@ process_request(request_context&& ctx, ss::smp_service_group g) {
         return do_process<delete_topics_api>(std::move(ctx), g);
     case describe_groups_api::key:
         return do_process<describe_groups_api>(std::move(ctx), g);
+    case sasl_handshake_api::key:
```

Comment:
as we start to add more cases here we should push to the top the most common cases (produce, consume)